subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractIntegrationSpec.groovy

```diff
@@ -487,7 +487,7 @@ tmpdir is currently ${System.getProperty("java.io.tmpdir")}""")
         failure = executer.withTasks(*tasks).runWithFailure()
 
         if (enableProblemsApiCheck && buildOperationsFixture.problems().isEmpty()) {
-            throw new AssertionFailedError("Expected to receive a problem event accompanying the build failure but did not.")
+            throw new AssertionFailedError("Expected problem to reported for the failed build, but none was reported.")
```

Comment:
maybe this? Expected to find a problem emitted via the 'Problems' service for all failing builds but none was received.

Reply:
```suggestion
            throw new AssertionFailedError("Expected to find a problem emitted via the 'Problems' service for all failing builds but none was received.")
```